patroni/utils.py

```diff
@@ -1064,6 +1064,60 @@ def data_directory_is_empty(data_dir: str) -> bool:
     return all(os.name != 'nt' and (n.startswith('.') or n == 'lost+found') for n in os.listdir(data_dir))
 
 
+def apply_keepalive_limit(option: str, value: int) -> int:
+    """
+    Adjusts the keepalive option value to ensure it falls within the acceptable range for the current operating system.
+
+    :param option: The TCP keepalive option name.
+        Possible values are "TCP_USER_TIMEOUT", "TCP_KEEPIDLE", "TCP_KEEPINTVL", and "TCP_KEEPCNT".
+    :param value: The desired value for the keepalive option.
+
+    Returns:
+    :returns: The adjusted value that falls within the acceptable range for the specified keepalive option.
+    """
+    min_possible_value = 1
+    max_possible_value = None
+    if sys.platform.startswith('linux'):
+        if option == "TCP_USER_TIMEOUT":
+            max_possible_value = 2147483647
+        if option == "TCP_KEEPIDLE":
+            max_possible_value = 32767
+        if option == "TCP_KEEPINTVL":
+            max_possible_value = 32767
+        if option == "TCP_KEEPCNT":
+            max_possible_value = 127
+
+    if sys.platform.startswith('darwin'):
+        if option == "TCP_KEEPIDLE":
+            max_possible_value = 4294967
+        if option == "TCP_KEEPINTVL":
+            max_possible_value = 4294967
+        if option == "TCP_KEEPCNT":
+            max_possible_value = 2147483647
+
+    if sys.platform.startswith('win32'):
+        if option == "TCP_KEEPIDLE":
+            max_possible_value = 2147483647
+        if option == "TCP_KEEPINTVL":
+            max_possible_value = 2147483647
+        if option == "TCP_KEEPCNT":
+            max_possible_value = 255
+
+    if max_possible_value is not None and value > max_possible_value:
+        logger.debug(
+            '%s changed from %d to %d.', option, value, max_possible_value
+        )
```

Comment:
It perfectly fits to 120 characters.
```suggestion
        logger.debug('%s changed from %d to %d.', option, value, max_possible_value)
```